<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kontrol Sende</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#AC7D88;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#85586F;
color: rgb(233, 255, 249);
border-radius:5px;
</string>
       </property>
       <property name="text">
        <string>Başlat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#85586F;
color: rgb(233, 255, 249);border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Duraklat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#85586F;
color: rgb(233, 255, 249);border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Sıfırla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMusic">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#DEB6AB;
color: rgb(233, 255, 249);border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Müzik</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTodo">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#DEB6AB;
color: rgb(233, 255, 249);border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Ajanda Aracı</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTracker">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#DEB6AB;
color: rgb(233, 255, 249);border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Başarı Takip Aracı</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#D6EFED;</string>
    </property>
    <property name="title">
     <string>Süre Ayarla</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignTop">
       <widget class="QRadioButton" name="rdWork">
        <property name="font">
         <font>
          <family>Bahnschrift Condensed</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Çalışma</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QRadioButton" name="rdShortBreak">
        <property name="font">
         <font>
          <family>Bahnschrift Condensed</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kısa Mola</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdLongBreak">
        <property name="font">
         <font>
          <family>Bahnschrift Condensed</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Uzun Mola</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="kalanPomodoro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>701</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#D6EFED;</string>
    </property>
    <property name="text">
     <string>Kontrol Sende!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Condensed</family>
      <pointsize>72</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#D6EFED;</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
